### PR TITLE
Introduce StyleCop.Analyzers

### PR DESCRIPTION
**Pull request by moreal**
Applied `StyleCop.Analyzers` and update codes to follow the rule.
Also fixed language version to *7.2*.

### CODE REVIEW

File: Bencodex.Tests.ruleset [hunk 1]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet
+        Name="Rules for Libplanet"
+        Description="Code analysis rules for Libplanet.csproj."
```
Comment (dahlia) [inline Bencodex.Tests.ruleset]: 👀

File: Bencodex.ruleset [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet
+        Name="Rules for Libplanet"
+        Description="Code analysis rules for Libplanet.csproj."
```
Comment (dahlia) [inline Bencodex.ruleset]: :eyes:

File: Bencodex.Tests.ruleset (hunk 1 above)
Comment (moreal) [reply Bencodex.Tests.ruleset]: 🙄

File: Bencodex.Tests.ruleset [hunk 3]
```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RuleSet
+        Name="Rules for Bencodex.Test"
+        Description="Code analysis rules for Bencodex.Test.csproj."
```
Comment (dahlia) [inline Bencodex.Tests.ruleset]: The project name should be plural here, right?

Comment (moreal) [reply Bencodex.Tests.ruleset]: Yeah, I will fix it.

**Pull request merged**